### ISSUE TITLE
Rollerball not moving during training "No episode was completed since last summary. Training." - mlagents-2.2.1 - py=3.7

### ISSUE THREAD

**Issue by kryptoish** [Unity-Technologies/ml-agents#5798]
I built a learning environment with ml-agents in unity following this tutorial:
https://github.com/Unity-Technologies/ml-agents/blob/main/docs/Learning-Environment-Create-New.md
After following all instructions on how to create the environment, I wanted to train it. It did not work after I started the simulation so I made a fix to the yaml file using this solution:
https://github.com/Unity-Technologies/ml-agents/issues/3957
After that the training started to work but it did not start moving and the training files outputted cannot be used afterwards. 
This is the YAML file used:

default:
    trainer: ppo
    batch_size: 10
    beta: 5.0e-4
    buffer_size: 100
    epsilon: 0.2
    hidden_units: 128
    lambd: 0.99
    learning_rate: 3.0e-4
    learning_rate_schedule: linear
    max_steps: 500000
    memory_size: 128
    normalize: false
    num_epoch: 3
    num_layers: 2
    time_horizon: 64
    sequence_length: 64
    summary_freq: 10000
    use_recurrent: false
    vis_encode_type: simple
    reward_signals:
        extrinsic:
            strength: 1.0
            gamma: 0.99

RollerBall:
    batch_size: 10
    buffer_size: 100
    memory_size: 128

The prompt that the training was running on kept on showing the following out put when training:
![image](https://user-images.githubusercontent.com/69884464/194973474-1809cfc7-17da-4fd1-978a-51af65d8dc69.png)

This is the code below
[RollerBall.zip.zip](https://github.com/Unity-Technologies/ml-agents/files/9750040/RollerBall.zip.zip)

Asset file:
[Rollerballerscene.zip.zip](https://github.com/Unity-Technologies/ml-agents/files/9750044/Rollerballerscene.zip.zip)

Any help would be much appreciated. Thank you!

**Comment by Eelam** [comment]
Increase the 'summary_freq: 10000' e.g. 'summary_freq: 50000'

**Comment by kryptoish** [comment]
> Increase the 'summary_freq: 10000' e.g. 'summary_freq: 50000'

Did not change anything :( 
![image](https://user-images.githubusercontent.com/69884464/197308763-64b281bb-0f9b-4d7b-b80b-2a81d060418f.png)

**Comment by Eelam** [comment]
Hi,
I followed the: [https://github.com/Unity-Technologies/ml-agents/blob/main/docs/Learning-Environment-Create-New.md](url)
I have attaced a version 'RollerBall' that works on my system. You should be able to import it and test it with the VRollerBall.yaml.

If it does not work on your system, then the problem is on the ml-agents installation. Did you test any of the example: [https://github.com/Unity-Technologies/ml-agents/blob/main/docs/Learning-Environment-Examples.md](url)

Why haven’t you provided the Environment info as the template states?

My Environment

- Unity Version: Unity 2020.3.33f1
- OS + version: Windows 11
- ML-Agents version: ML-Agents v0.29.0
- Torch version: 1.9.1+cu111
- Environment: Python 3.8.10

**Note**: 'cu111' for Nvidia GPU

[VRollerBall.zip](https://github.com/Unity-Technologies/ml-agents/files/9847212/VRollerBall.zip)

**Comment by kryptoish** [comment]
> 

Thank you!
The problem I was having was that my ml-agents version was set to an older version in the venv that I was running it.
Also sorry for not including my Environment info, I will do so next time.

My env:
Unity Version: Unity 2021.3.8f1
OS + version: Windows 10 pro
ML-Agents version: ML-agents v0.29.0 (I was using v0.16.0 by accident (dont do this))
Torch version: 1.12.1
Environment: Python 3.7.13

Also the links that you have included don't exist or don't work because I couldn't find them.
